docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+# Creating a channel
+
+In order to create and transfer assets on a Hyperledger Fabric network, an organization needs to join a channel. Channels are a private layer of communication between specific organizations. Each channel consists of a separate ledger that can only be read and written to by channel members, who are allowed to join their peers to the channel and receive new blocks of transactions from the ordering service. While the peers, nodes, and Certificate Authorities form the physical infrastructure of the network, channels are the process by which organizations connect with each other and interact. Channels are invisible to other members of the network, making them the most powerful tool for privacy in a blockchain consortium.
```

Comment:
```
Channels are invisible to other members of the network, making them the most powerful tool for privacy in a blockchain consortium.
```
I don't think we want to say this ... private data collections are actually the **most powerful** construct we have for privacy.

Reply:
Will remove and leave the sentence as `Channels are invisible to other members of the network.`